src/Umbraco.Web/Mvc/JsonNetResult.cs

```diff
@@ -22,10 +22,19 @@ public class JsonNetResult : ActionResult
         public JsonSerializerSettings SerializerSettings { get; set; }
         public Formatting Formatting { get; set; }
 
+        /// <summary>
+        /// Default, unchanged JsonSerializerSettings
+        /// </summary>
+        public static readonly JsonSerializerSettings DefaultJsonSerializerSettings = new JsonSerializerSettings();
+
         public JsonNetResult()
         {
             SerializerSettings = new JsonSerializerSettings();
```

Comment:
Should this change to `SerializerSettings  = DefaultJsonSerializerSettings` ?

Reply:
Not if you change the value of DefaultJsonSerializerSettings.